gobblin-data-management/src/main/java/gobblin/data/management/copy/CopyableFile.java

```diff
@@ -86,10 +89,17 @@
   /** Timestamp of file as in upstream. */
   private long upstreamTimestamp;
 
-  @lombok.Builder(builderClassName = "Builder", builderMethodName = "_hiddenBuilder")
   public CopyableFile(FileStatus origin, Path destination, OwnerAndPermission destinationOwnerAndPermission,
       List<OwnerAndPermission> ancestorsOwnerAndPermission, byte[] checksum, PreserveAttributes preserve,
       String fileSet, long originTimestamp, long upstreamTimestamp, Map<String, String> additionalMetadata) {
+    this(origin, destination, destinationOwnerAndPermission, ancestorsOwnerAndPermission, checksum, preserve, fileSet,
+        originTimestamp, upstreamTimestamp, additionalMetadata, "");
+  }
+
+  @lombok.Builder(builderClassName = "Builder", builderMethodName = "_hiddenBuilder")
```

Comment:
The point of the builder pattern is to not have to create a new constructor when we add a new parameter (as long as all accesses are through a builder). I think the above constructor can be removed.